addon/components/layout/side-hover-panel/component.js

```diff
@@ -0,0 +1,63 @@
+import Ember from 'ember';
+import layout from './template';
+let $ = Ember.$;
+
+export default Ember.Component.extend({
+  layout,
+  classNames: ['__side-hover-panel'],
+  
+  // Those are the parameters that you can override
+  backdropAction: null,
+  side: "right",
+  shouldAnimate: "true",
+  width: "100%",
+  height: "100%",
+  disableScrolling: "false",
+  stickTo: "right",
+
+  didInsertElement() {
+    // limit the amount of work for the brower
+    this.$().detach().prependTo($(document.body));
+    this.$('.hover-panel').addClass(this.get('side') + "_side");
+    this.$('.hover-panel').addClass(this.get('stickTo') + "_align");
+
+    // Set animations
+    if (this.get('shouldAnimate') === "true")
```

Comment:
I didn't find any good way to pass `bool` in a handlebar helper
Doing this would force the usage to
```
shouldAnimate="false" is {{if stringBool "true" "false"}}
```